src/pages/api/github/pr.ts

```diff
@@ -108,28 +136,39 @@ async function createForkAndPR(
   // Create new file on the branch
   // const _trimmedFileName = fileName.trim();
   const fileSlug = deriveFileSlug(fileName);
-  await octokit.request("PUT /repos/:owner/:repo/contents/:path", {
-    owner: forkOwner,
-    repo: forkRepo,
-    path: `${directoryPath}/${fileSlug}.md`,
-    message: `Added "${metaData.fileTitle}" transcript submitted by ${forkOwner}`,
-    content: fileContent,
-    branch: newBranchName,
-  });
+  const updateContent = await octokit.request(
+    "PUT /repos/:owner/:repo/contents/:path",
+    {
+      owner: forkOwner,
+      repo: forkRepo,
+      path: `${directoryPath}/${fileSlug}.md`,
+      message: `${prUrl ? "Updated" : "Added"} "${
```

Comment:
<img width="718" alt="Screenshot 2023-10-18 at 4 27 16 PM" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/aee4cd99-c05c-4fb5-8119-f4baaec4132f">

The updated line is really gross with the "" "" next to each other